adventureGame/AdventureGame/src/Main.py

```diff
@@ -28,17 +43,51 @@ def introduction():
     time.sleep(3)
     print("Sir Gregos IV: \"The names Sir Gregos IV, long serving commander of the king\'s forces\"\n")
```

Comment:
Change print statement to:
print("Sir Gregos IV: \"The name's Sir Gregos IV, long serving commander of the king\'s forces\"\n")

Reply:
Fixed.